docs/general/server/media/shows.md

```diff
@@ -68,6 +68,16 @@ Episode numbering for specials may vary from metadata provider to metadata provi
 
 :::
 
+Specials can also be shown within a Season if so desired. This can be helpful when they are part of a continued storyline during the Season. This requires 2 settings:
```

Comment:
```suggestion
Specials can also be shown within a season if so desired. This can be helpful when they are part of a continued storyline during the season. This requires 2 settings:
```